config/config.toml.example

```diff
@@ -451,3 +451,9 @@ allow-expression-index = false
 [isolation-read]
 # engines means allow the tidb server read data from which types of engines. options: "tikv", "tiflash", "tidb".
 engines = ["tikv", "tiflash", "tidb"]
+
+# set labels for the tidb server. The labels describe some specific properties for the tidb server
```

Comment:
I prefer to hide this comment to prevent the user from using it.

Reply:
we can't hide this config because we need tidb-operator to use it.

Reply:
updated.